### ISSUE TITLE
What to do with gratuated/left collegists?

### ISSUE THREAD

**Issue by kdmnk** [luksan47/mars#319]
Delete them?

**Comment by luksan47** [comment]
Yes, probably that will be the solution. We will be able to solve it automatically when we set up the status statements and status handling at the end of the year.

**Comment by luksan47** [comment]
Actually, we can set them as inactive. It's good to record the alumni somewhere and probably that's an ask from the secretariat as well.
